doc/news/svg-from-img.rst

```diff
@@ -0,0 +1,3 @@
+**Added:**
+
+* Added an option in the CLI to create linked SVG file from image directly.
```

Comment:
```suggestion
* Added the option `create-svg` in the CLI to create an SVG with an embedded or linked PNG or JPEG directly from an image.
```

Reply:
is there already an option here to choose between linked and embedded?

Reply:
Should we really add this option for the user directly?